### ISSUE TITLE
Timex.format drops leading zeros in milliseconds

### ISSUE THREAD

**Issue by linges** [bitwalker/timex#461]
### Steps to reproduce

` ~N[2018-08-17 16:11:23.054] |> Timex.format!("{ISO:Extended:Z}") `
Result     : `"2018-08-17T16:11:23.540Z"`
Excepted: `"2018-08-17T16:11:23.054Z"`
### Description of issue
I used Timex.format to convert naive timestamps to strings but notice that the leading zero in the milliseconds get dropped, which oblivious changes the timestamp.

**Comment by mdepolli** [comment]
Seems to be fixed in 3.3.0.

```
iex(1)> ~N[2018-08-17 16:11:23.054] |> Timex.format!("{ISO:Extended:Z}")
"2018-08-17T16:11:23.054Z"
```

**Comment by ckhrysze** [comment]
@linges Thank you for reporting! As this issue is fixed in both the latest release (3.3.0) and still fine in master, I'm closing this issue.
